apache/couchdb-fauxton
# Update documents/designdocinfo to use redux

## Overview

Update documents/designdocinfo to use redux

## Testing recommendations

- Go to Database > Design Doc > Metadata
  - Verify the ddoc metadata is displayed
- Select another ddoc metadata
  - Verify the page is updated with metadata from the 2nd ddoc

## Checklist
